azext_edge/edge/providers/adr/_help.py

```diff
@@ -807,6 +807,10 @@ def load_iotops_adr_help():
         - name: Add a rest endpoint with authentication
           text: >
             az iot ops ns device endpoint inbound add rest --device mydevice --instance myInstance -g myInstanceResourceGroup --name myEndpoint --endpoint-address "https://api.example.com/data" --user-ref usernameSecret --pass-ref passwordSecret
+
+        - name: Add a rest endpoint with certificate authentication
+          text: >
+            az iot ops ns device endpoint inbound add rest --device mydevice --instance myInstance -g myInstanceResourceGroup --name myEndpoint --endpoint-address "https://api.example.com/data" --cert-ref certificateSecret
```

Comment:
nit: @digimaun looks like the CLI just expects the user to pass the secret reference string in the endpoint auth fields (username, password, etc.) correct? These content of all these secret fields for device auth are expected to be in the form `<k8s-secret-name>/<secret-key>`. Maybe it would be worth mentioning it in the helper text for the auth secret params.